src/main/java/com/practise/tdd/SpecialDigitsTextMapper.java

```diff
@@ -41,6 +44,26 @@ public String getValue(Integer number) {
 		return numberInWords.toString();
 	}
 
+	private void appendThousandthDigitValue(Integer number, StringBuilder numberInWords) {
```

Comment:
Method length reduced. Readability is better if seen in Unified view instead of Split, when viewing differences.